README.md

```diff
@@ -92,6 +92,27 @@ Pylance provides users with the ability to customize their Python language suppo
         - `true`
         - `false`
 
+-   `python.analysis.autoImportCompletions`
+
+    -   Used to control the offering of auto-imports in completions.
+    -   Accepted values:
+        -   `true` (default)
+        -   `false`
+
+-   `python.analysis.packageIndexDepths`
+
+    -   Used to control how many installed packages indexer will index.
```

Comment:
maybe "Used to control how many subfolders under installed packages should be indexed." or something like that? Just to reflect what depth would mean.

Reply:
so, depth is not subfolder but nested modules. simply put, number of ```dot``` + 1

ex) ```package\__init``` is basically ```import package``` so it is depth 1
```package\lib.py``` is ```import package.lib``` so it is depth 2 even if it is on same folder?

not sure how to express it...

Reply:
"Used to override the depth of indexing on a per package basis. Depth is measured in blah"?

Reply:
nice. it would be even better if you can provide the ```blah``` part as well :)

Reply:
"Used to override the depth of installed package indexing on a per package basis. By default, only top-level modules are indexed (depth = 1). To index submodules, increase depth by 1 for each level of submodule you want to index."

I'm not totally satisfied with this. Maybe someone else can improve it. :)

Reply:
I actually like it a lot. I think the example below helps illustrate what this means.